packages/slice-machine/lib/models/common/widgets/ContentRelationship/index.ts

```diff
@@ -26,15 +27,15 @@ const Meta = {
   description: "Define content relations & internal links",
 };
 
-const configSchema = yup.object().shape({
+const contentRelationShipConfigSchema = linkConfigSchema.shape({
```

Comment:
Calling a second `shape` here acts like an `Object.assign` on the `linkConfigSchema` right?

Reply:
Seems like it does, to be tested.
```ts
function shape<TNextShape extends ObjectShape>(additions: TNextShape, excludes?: [string, string][]): ObjectSchema<Assign<TShape, TNextShape>, TContext, TypeOfShape<Assign<TShape, TNextShape>> | Optionals<TIn>>;
```

Reply:
yes this is the case, here the link to understand how the inheritance works with yup https://stackoverflow.com/questions/68128636/react-yup-formik-how-to-extend-current-schema-from-another-schema